### ISSUE TITLE
[Bug] Pydantic error on import

### ISSUE THREAD

**Issue by stevepepple** [mendableai/firecrawl#929]
**Describe the Bug**

When first installing and running the project on python3.11, I get this error on import: 

```
NameError: Field name "schema" shadows a BaseModel attribute; use a different field name with "alias='schema'".
```

**To Reproduce**
Steps to reproduce the issue:
1. Install with pip
2. Import the library in runtime. I did the following: 

```
ipython
Then
from firecrawl import FirecrawlApp
```

Look like installing v 1.5.0 doesn't have the same issue. 

**Expected Behavior**

**Screenshots**
If applicable, add screenshots or copies of the command line output to help explain the issue.

**Environment (please complete the following information):**
- OS: [e.g. macOS, Linux, Windows]
- Firecrawl Version: [e.g. 1.2.3]
- Node.js Version: [e.g. 14.x]

**Logs**
If applicable, include detailed logs to help understand the problem.

**Additional Context**
Add any other context about the problem here, such as configuration specifics, network conditions, data volumes, etc.

**Comment by Korred** [comment]
Just came across the same (?) issue on import.

`E:\Programming\miniconda3\envs\ai_devs3\Lib\site-packages\pydantic\_internal\_fields.py:172: UserWarning: Field name "schema" in "FirecrawlApp.ExtractParams" shadows an attribute in parent "BaseModel"
  warnings.warn(`




**Environment:**

OS: Windows
Firecrawl Version: 1.6.1
Python Version: 3.12.7

**Comment by nmathew87** [comment]
Ran into the same issue

**Comment by Korred** [comment]
@rafaelsideguide relevant: https://github.com/pydantic/pydantic/issues/242

**Comment by kyle-alm** [comment]
In firecrawl.py, line 30, replace the field `schema` and then provide that as an alias. Here's the change I made:

```
    class ExtractParams(pydantic.BaseModel):
        """
        Parameters for the extract operation.
        """
        prompt: str
        extract_schema: Optional[Any] = pydantic.Field(None, alias="schema")
        system_prompt: Optional[str] = None
        allow_external_links: Optional[bool] = False
```

**Comment by nickscamara** [comment]
Fixed on 1.16.4!
